src/Sylius/Behat/Page/Shop/Product/ShowPage.php

```diff
@@ -165,7 +164,7 @@ public function getCurrentUrl(): string
 
     public function getCurrentVariantName(): string
     {
-        $currentVariantRow = $this->getElement('current_variant_input')->getParent()->getParent();
+        $currentVariantRow = $this->getElement('current_variant_input')->getParent()->getParent()->getParent();
```

Comment:
```suggestion
        $currentVariantRow = $this->getElement('current_variant_input')->getParent()->getParent()->getParent();
```
isn't there any way to get it directly?

Reply:
This need some changes on frontend, so probably it will change anyway